.github/workflows/stale.yml

```diff
@@ -15,9 +15,9 @@ jobs:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         stale-issue-message: 'This issue is stale because it has been open 60 days with no activity. Remove stale label or comment or this will be closed in 15 days'
         stale-pr-message: 'This pr is stale because it has been open 60 days with no activity. Remove stale label or comment or this will be closed in 15 days'
-        days-before-stale: 60
-        days-before-close: 15
+        days-before-stale: 90
+        days-before-close: 21
         stale-issue-label: 'lifecycle/stale'
         stale-pr-label: 'lifecycle/stale'
-        exempt-issue-label: 'lifecycle/frozen'
-        exempt-pr-label: 'lifecycle/frozen'
+        exempt-issue-label: 'lifecycle/frozen,type/enhancement,type/suggestion,type/bug'
```

Comment:
Some reported issues labeled `bug` are actually not bugs, maybe configuration or environment problems. I feel we shouldn't include it here.

Reply:
If we can confirm the reported issue is not a bug, we should remove the `bug` label.

Reply:
From my experience, this is not an easy thing and it is prone to forget. If we want to enable this we need to have regular bug scrub to revisit issues. If we can ensure removing `bug` label for every wrong issue then it sounds good.